js/wp-seo-admin-306.js

```diff
@@ -81,10 +81,29 @@ jQuery( document ).ready( function() {
 		jQuery( '#' + activeTab + '-tab' ).addClass( 'nav-tab-active' );
 
 		jQuery( '.nav-tab-active' ).click();
+
+		initSelect2();
 	}
 );
 
 /**
+ * Adds select2 for selected fields.
+ */
+function initSelect2() {
```

Comment:
This might conflict errors when the this file and the metabox js are both loaded.

The function in `wp-seo-metabox-302.js` is initialized in a isolated scope, but this one is the global scope, this can cause conflicts.